config/initializers/read_only_mode.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Orangelight
+  def read_only_mode
+    @read_only_mode ||= ENV.fetch("ORANGELIGHT_READ_ONLY_MODE", false) == "true"
+  end
+
+  def read_only_message
+    default_msg = "Orangelight is in read-only mode."
```

Comment:
Probably should say "The Catalog" instead of "Orangelight"

Reply:
Yep. Updated.